project/plugins.sbt

```diff
@@ -21,11 +21,11 @@ resolvers += "HMRC Releases" at "https://dl.bintray.com/hmrc/releases"
 
 resolvers += Resolver.typesafeRepo("releases")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.10.0")
-addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.1.0")
-addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "1.5.0")
-addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.0.0")
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.20")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.13.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.2.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "1.13.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.1.0")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.10")
```

Comment:
is sbt-plugin being downgraded here?

Reply:
good spot, I originally updated to the 2.7 version which caused all sorts of issues, reverted but looks like I typed in wrong version!  Will sort out and run all tests again with correct version